### ISSUE TITLE
Wrong and missing match equivalents

### ISSUE THREAD

**Issue by ratoaq2** [Diaoul/subliminal#679]
Looking at the matching equivalencies, I found [this](https://github.com/Diaoul/subliminal/blob/master/subliminal/score.py#L107):
```
        if 'tvdb_id' in matches:
            logger.debug('Adding tvdb_id match equivalents')
            matches |= {'series', 'year'}
```

Since `tvdb_id` is the episode tvdb_id and `series_tvdb_id` is the series tvdb_id, I believe that the correct would be:
```
        if 'tvdb_id' in matches:
            logger.debug('Adding tvdb_id match equivalents')
            matches |= {'series', 'year', 'season', 'episode'}
        if 'series_tvdb_id' in matches:
            logger.debug('Adding series_tvdb_id match equivalents')
            matches |= {'series', 'year'}
```

And just as an additional note, after analyzing thousands of release names from episodes, I noticed these standards:
 - The vast majority of the release names contains season and episode
 - A minor part of them contains an `absolute episode number` or `parts` (sometimes with season, sometimes with an year) - Probably that should be also equivalent to season + episode match
 - Some few cases you have only air date (probably air date match should be equivalent to season + episode match)

**Comment by Diaoul** [comment]
Fixed with f52ef6d44a1ca31ac7f8950e5cd687d2bbc88448

**Comment by Diaoul** [comment]
Thanks
